config/environment.js

```diff
@@ -82,10 +82,10 @@ module.exports = function (environment) {
   }
 
   if (environment === 'production') {
-    ENV.apiHost = 'apiHOST';
-    ENV.wholeTaleHost = 'https://dashboardHOST';
-    ENV.dataOneHost = 'dataOneHOST';
-    ENV.authProvider = 'authPROVIDER';
+    ENV.apiHost = 'https://girder.dev.wholetale.org';//'apiHOST';
+    ENV.wholeTaleHost = 'https://dashboard.dev.wholetale.org';//'https://dashboardHOST';
+    ENV.dataOneHost = 'https://dev.nceas.ucsb.edu/knb/d1/mn/v2';//'dataOneHOST';
+    ENV.authProvider = 'Globus';//'authPROVIDER';
```

Comment:
I think `production` should stay `production` ;-) Could you please revert that?

Reply:
Yes, sorry about that. It lingered from testing some other changes.

Reply:
Fixed with commit [#0a485c9](https://github.com/whole-tale/dashboard/pull/325/commits/0a485c90c0225bd464c6c278322f37f193f5e5e9)